projects/addon-editor/extensions/image-editor/image-editor.component.less

```diff
@@ -0,0 +1,43 @@
+@import 'taiga-ui-local';
+
+@controller-width: 0.375rem;
+
+:host {
+    position: relative;
+    display: inline-block;
+
+    &:hover .controller {
+        opacity: 1;
+    }
+}
+
+.controller {
```

Comment:
Let's make it pseudo element for smaller DOM nesting?

Reply:
done